modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -804,7 +791,7 @@ definition {
 		task ("When the object entry is deleted") {
 			ObjectAdmin.goToCustomObject(objectName = "CustomObject178781");
 
-			ObjectPortlet.deleteEntry(entry = "Trigger");
+			ObjectPortlet.deleteEntry(entry = "Entry test");
```

Comment:
This change doesn't belong here, it should be taken care of in the commit that originally made the change. 
1. You shouldn't be fixing an error you made earlier in another commit, just fix the original commit
2. You shouldn't be adding changes that aren't part of the logical changes in the commit. Since you're fixing the API usage here, you shouldn't be renaming the entry unless it's part of switching to the API.